terraform/environments/cdpt-chaps/database.tf

```diff
@@ -9,15 +9,11 @@ resource "aws_db_instance" "database" {
   engine_version         = "14.00.3381.3.v1"
   instance_class         = local.application_data.accounts[local.environment].db_instance_class
   identifier             = local.application_data.accounts[local.environment].db_instance_identifier
-  skip_final_snapshot    = true
```

Comment:
Should this be removed. According to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance it will default to false.

Reply:
Yes, if it's removed, then skip=false, and the identifier will be "${local.application_data.accounts[local.environment].db_instance_identifier}-db-snapshot"